CHANGELOG.md

```diff
@@ -26,6 +30,15 @@ All notable changes to this project will be documented in this file.
 - Allow dynamic autoloading for classes added during upgrade
 - Re-open account settings
 - Overlapping of primary menu with recipient
+- Editing state on new message modal
```

Comment:
not a user facing change AFAIK

Reply:
the _go back_ button didnt work because of the editing state still existing on the template, so I rephrased the sentence.